### ISSUE TITLE
pystorm.log.path must be set?

### ISSUE THREAD

**Issue by tdhopper** [Parsely/streamparse#293]
I've been running my topology on 344ce8cbae7e3202876227e3942741d62d767250 and am trying to move to 3.0.0. However a8861e4af53a5236605424324927e59ce7711ca7 results in this exception on `sparse submit`: 

```
Traceback (most recent call last):
  File "/Users/tdhopper/miniconda/envs/dir/bin/sparse", line 11, in <module>
    sys.exit(main())
  File "/Users/tdhopper/miniconda/envs/dir/lib/python2.7/site-packages/streamparse/cli/sparse.py", line 56, in main
    args.func(args)
  File "/Users/tdhopper/miniconda/envs/dir/lib/python2.7/site-packages/streamparse/cli/submit.py", line 243, in main
    override_name=args.override_name)
  File "/Users/tdhopper/miniconda/envs/dir/lib/python2.7/site-packages/streamparse/cli/submit.py", line 208, in submit_topology
    env_config, nimbus_client, options=options)
  File "/Users/tdhopper/miniconda/envs/dir/lib/python2.7/site-packages/streamparse/cli/submit.py", line 72, in _submit_topology
    print("Routing Python logging to {}.".format(options['pystorm.log.path']))
KeyError: u'pystorm.log.path'
```

It goes away if I set the log path for my environment in `config.json`. However, I have it unset so python logs go to `worker.log`.

**Comment by dan-blanchard** [comment]
This is what I get for sneaking in last minute changes.  Will fix and push out 3.0.1 very soon.

**Comment by dan-blanchard** [comment]
Fixed and streamparse 3.0.1 is up on PyPI

**Comment by tdhopper** [comment]
Thanks @dan-blanchard. You do great work!
